mobile/native/Cargo.toml

```diff
@@ -10,4 +10,9 @@ crate-type = ["cdylib", "staticlib"]
 
 [dependencies]
 anyhow = "1"
+atty = "0.2"
 flutter_rust_bridge = "1"
+state = "0.5.3"
+time = { version = "0.3", features = ["serde", "parsing", "std", "formatting", "macros", "serde-well-known"] }
```

Comment:
Do we need all these features?

Reply:
Second question: can we use of `time` which is included in `tracing-subscriber`?